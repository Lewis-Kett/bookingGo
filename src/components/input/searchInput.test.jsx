import React from 'react';
import  SearchInput from './searchInput';
import renderer from 'react-test-renderer'

describe('searchInput', () => {

  it('should match snapshot', () => {

    const tree = renderer.create(<SearchInput placeholder='test' value={''} onChange={()=> {}} loaded={true} focused={()=> {}} blur={()=> {}}/>).toJSON()

    expect(tree).toMatchSnapshot()
  });

})
