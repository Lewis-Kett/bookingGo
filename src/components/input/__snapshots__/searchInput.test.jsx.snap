// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchInput should match snapshot 1`] = `
<div
  className="inputWrapper"
>
  <input
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    placeholder="test"
    type="text"
    value=""
  />
  <span />
</div>
`;
